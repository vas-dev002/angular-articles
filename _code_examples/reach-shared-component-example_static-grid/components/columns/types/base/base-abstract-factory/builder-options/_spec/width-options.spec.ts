import { WidthOptions } from "..";

declare const expect: jest.Expect;

describe("[Option]: WidthOptions", () => {
	let widthOptions: WidthOptions;

	beforeEach(() => {
		widthOptions = new WidthOptions();
	});

	describe("[WidthOptions instantiate]", () => {
		it("should be successfully created", () => {
			expect(widthOptions).toBeTruthy();
			expect(widthOptions).toMatchSnapshot();
		});
	});
});
