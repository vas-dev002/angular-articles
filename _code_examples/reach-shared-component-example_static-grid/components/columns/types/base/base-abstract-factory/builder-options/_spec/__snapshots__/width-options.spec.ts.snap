// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Option]: WidthOptions [WidthOptions instantiate] should be successfully created 1`] = `
WidthOptions {
  "maxWidth": 500,
  "minWidth": 50,
}
`;
